modules/bbp-simulation/src/main/resources/schemas/hbphackaton/simulation/modelvalidation/v0.1.0.json

```diff
@@ -0,0 +1,55 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/core/schema/v0.1.0",
+    {
+      "nsg": "https://bbp-nexus.epfl.ch/vocabs/bbp/neurosciencegraph/core/v0.1.0/",
+      "this": "{{base}}/schemas/hbphackaton/simulation/modelvalidation/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/bbp/core/activity/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:ModelValidationShape",
+      "@type": "sh:NodeShape",
+      "label": "Model validation shape",
+      "targetClass": "nsg:ModelValidation",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/bbp/core/activity/v0.1.0/ActivityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "prov:used",
+              "name": "Model",
+              "description": "Used model.",
+              "seeAlso": "{{base}}/schemas/hbphackaton/simulation/modelinstance/v0.1.0/shapes/ModelInstanceShape",
+              "class": "nsg:ModelInstance",
+              "minCount": 1
+            },
+            {
+              "path": "prov:used",
+              "name": "Model",
+              "description": "Used model.",
+              "seeAlso": "{{base}}/schemas/hbphackaton/simulation/modelinstance/v0.1.0/shapes/ModelInstanceShape",
+              "class": "nsg:VariableReport",
+              "minCount": 1
```

Comment:
Do we need to use sh:qualifiedValueShapesDisjoint for these two prov:used property shapes?